JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -311,8 +330,20 @@ fun Markup.toAnnotatedStringItem(typography: Typography): AnnotatedString.Item<S
     }
 }
 
-@Preview
+private fun ColorPalette.codeBlockBackground() = onSurface.copy(alpha = .15f)
```

Comment:
Could this be an ext property instead of an ext fun?

Reply:
Sure